src/qos_client/src/yubikey.rs

```diff
@@ -29,6 +29,7 @@ pub const SIGNING_SLOT: SlotId = SlotId::Signature;
 /// Factory default pin for yubikeys.
 pub const DEFAULT_PIN: &[u8] = b"123456";
 const ALGO: AlgorithmId = AlgorithmId::EccP256;
+const VALIDITY_SECS: u32 = 60 * 60 * 24 * 365;
```

Comment:
1 year. Purposefully using an smaller integer because of the `u64::MAX` incident.